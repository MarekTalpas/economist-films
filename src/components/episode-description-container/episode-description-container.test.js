// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import EpisodeDescriptionContainer from './episode-description-container';

test('EpisodeDescriptionContainer renders correctly', () => {
  const tree : string = renderer.create(<EpisodeDescriptionContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});