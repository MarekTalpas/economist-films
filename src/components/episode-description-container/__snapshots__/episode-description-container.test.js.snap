exports[`test EpisodeDescriptionContainer renders correctly 1`] = `
<div
  className="episode-description">
  <h3
    className="episode-description__date">
    ...loading...
  </h3>
  <h1
    className="episode-description__title">
    ...loading...
  </h1>
  <p
    className="episode-description__text">
    ...loading...
  </p>
</div>
`;
